build/almalinux-9/include/install-build-dependencies.sh

```diff
@@ -169,6 +169,10 @@ dnf install -y python3-pip
 curl -L -o /usr/local/bin/factory-command "https://raw.githubusercontent.com/vespa-engine/vespa/refs/heads/master/.buildkite/factory-command.sh"
 chmod 755 /usr/local/bin/factory-command
 
+# Install helm for package management in Kubernetes
+chmod 700 get_helm.sh
+./get_helm.sh --version 3.17.0
```

Comment:
Nitpick, let's follow pattern of the rest, use the "include" path instead of relative:
```suggestion
/include/get_helm.sh --version 3.17.0
```

Reply:
Ah oops, too much multitasking :sweat_smile: fixed!

Reply:
`chmod` needs update too, but that could be just solved with git chmod? 😄